draft-ietf-quic-recovery.md

```diff
@@ -496,7 +496,7 @@ data.
 
 The initial probe timeout for a new connection or new path SHOULD be
 set to twice the initial RTT.  Resumed connections over the same network
-SHOULD use the previous connection's final smoothed RTT value as the resumed
+MAY use the previous connection's final smoothed RTT value as the resumed
```

Comment:
This doesn't have to do anything with connection IDs, does it?

Reply:
Oops, some stray commits ended up in there.